packages/uiweb/src/lib/components/space/SpacesUI.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import SpaceBanner, { SpaceBannerProps } from './SpaceBanner';
+import SpaceFeed, { SpaceFeedProps } from './SpaceFeed';
+import SpaceWidget, { SpaceWidgetProps } from './SpaceWidget';
+import SpaceTrendingList, { SpaceTrendingListProps } from './SpaceTrendingList';
```

Comment:
suggestion: can move all the exports to a single `index.ts` file and export all from a single source of truth.